_includes/js/leaflet-awesome-markers.js

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
Is there a compressed/minimized version of this?

Reply:
yes (I believe this is what the `.min.js` file is.